src/api/user.controller.js

```diff
@@ -0,0 +1,78 @@
+const { userService } = require('../services/user.service.js');
+
+const getUsers = async (req, res) => {
+  const users = await userService.getUsers();
```

Comment:
Consider adding error handling for the `userService.getUsers` call to manage any unexpected errors that might occur during the database query.